### DISCUSSION TITLE
restapi from go

### DISCUSSION THREAD

**Discussion by fdefilippo** [drakkan/sftpgo#590] [General]
Hi, is there any example in go of using the REST-api?

thnx

**Comment by drakkan** [comment]
Hi,

you can find some examples [here](https://github.com/drakkan/sftpgo/blob/main/httpdtest/httpdtest.go), this code is used for test cases so you should remove some test specific part but it should be enough to get started. 

You can also auto generate the Go code from the API definition, several OpenAPI generators are available, for example:

https://github.com/swagger-api/swagger-codegen
https://openapi-generator.tech/
